config/production/config.yaml

```diff
@@ -0,0 +1,2 @@
+baseurl: 'https://aegisub.github.io/aegisite/'
```

Comment:
Does this need to be changed if I want to primary deploy target to be aegisub.org?

Reply:
We can probably just turn [canonifyURLs](https://gohugo.io/content-management/urls/#canonicalization) off and remove the baseurl.
This seemes to be the old default from doks which has since been changed.

Reply:
But if you want to deploy this site to github pages, you should set `baseurl`. Otherwise, `baseurl` will be `https://aegisub.github.io`, which lead all links to be 404.

Reply:
Right Github Pages would only host on `https://<organization>.github.io` if the repo were named `<organization>.github.io`, otherwise it will host on `https://<organization>.github.io/<repo-name>`.

I talked with CoffeeFlux on Discord and he wants to host the `aegisub.org` site on his own server instead of Github infrastructure, but would be fine with having Github Pages up as a backup.

I guess we could add two build enviroments with seperate [config files](https://gohugo.io/getting-started/configuration/#configuration-directory) and [npm run scripts](https://github.com/Aeg-dev/AegiSite/blob/switch2hugo/package.json#L19-L20) (`--environment github`) to deploy both to Github Pages with canonicalized URLs and to CoffeeFlux' server without.
The alternative would be dropping Github Pages entirely.